dashboard/app/bisect_test.go

```diff
@@ -276,6 +313,39 @@ https://goo.gl/tpsmEJ#testing-patches`,
 		})
 	}
 
+	// BisectFix #4
+	pollResp = c.client2.pollJobs(build.Manager)
+	c.expectNE(pollResp.ID, "")
+	c.expectEQ(pollResp.Type, dashapi.JobBisectFix)
+	c.expectEQ(pollResp.ReproOpts, []byte("repro opts 4"))
+	jobID = pollResp.ID
+	done = &dashapi.JobDoneReq{
+		ID:          jobID,
+		Build:       *build,
+		Log:         []byte("bisectfix log 4"),
+		CrashTitle:  "bisectfix crash title 4",
+		CrashLog:    []byte("bisectfix crash log 4"),
+		CrashReport: []byte("bisectfix crash report 4"),
+		Commits: []dashapi.Commit{
+			{
+				Hash:       "46e65cb4a0448942ec316b24d60446bbd5cc7827",
+				Title:      "kernel: add a fix",
+				Author:     "author@kernel.org",
+				AuthorName: "Author Kernelov",
+				CC: []string{
+					"reviewer1@kernel.org", "\"Reviewer2\" <reviewer2@kernel.org>",
+					// These must be filtered out:
+					"syzbot@testapp.appspotmail.com",
+					"syzbot+1234@testapp.appspotmail.com",
+					"\"syzbot\" <syzbot+1234@testapp.appspotmail.com>",
+				},
+				Date: time.Date(2000, 2, 9, 4, 5, 6, 7, time.UTC),
+			},
+		},
+	}
+	done.Build.ID = jobID
+	c.expectOK(c.client2.JobDone(done))
```

Comment:
I don't see anything that prevents the app from sending results of fix bisections, but in this test we don't receive the email with results. What am I missing? Why there is no email?

Reply:
I just noticed that when there is a call to pollEmailBug() from the test we get the following:
```ERROR: failed to report job: failed to execute mail_bisect_result.txt template: template: mail_bisect_result.txt:1:17: executing "mail_bisect_result.txt" at
<.BisectCause.Commit>: can't evaluate field Commit in type *dashapi.BisectResult.
```
I have added a note to the test and modified pollCompletedJobs() to restrict BisectFix reporting.